frontend/src/pages/HomePage.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from "react";
+import GuessInput from "../components/GuessInput";
+import GuessResult from "../components/GuessResult.js";
+import useGuessChecker from "../hooks/useGuessChecker";
+
+const HomePage = () => {
+  const [finalGuessName, setFinalGuessName] = useState("");
+  const [letterResults, guessSubmitted] = useGuessChecker(finalGuessName);
```

Comment:
So it's very cool that you are building your own hooks, however I would say that in this case this is overboard, and there's a way of doing it that is a little more efficient/understandable. 

How I would personally do it would be to create a new function within home page like this:
```js
function setAndCheckGuess(guess) {
  setFinalGuessName(guess)
  // Move the check logic from the useEffect within useGuessChecker here
}
```

You can then easily pass in this function to GuessInput, in place of `setFinalGuessName`(A great example of why we made `GuessInput` "dumb" in the first place!). Then you can move the 2 `useState`s from `useGuessChecker` up here as well.

Reply:
- [ ] Look into deprecating the custom hook

Reply:
I did some moving of thangs.  Let me know if this is what you were thinking.  Happy to jump on a call too!

- [ ] Reviewers to confirm refactoring